chatbot/README.md

```diff
@@ -146,6 +157,22 @@ You can then click on the order to have a new component popping up:
 At this point the chatbot is up and running!
 
 
+### Docker
```

Comment:
I'd prefer grouping the `Docker` section with the `Deploy the application` section to keep things together if they're related (for production deployments). 
If the `Docker` section is meant for production, lets rename it to `Deploy for production`, similar to the Reference Storefront's layout.

Reply:
Fixed